Source/TextToSpeechV1/Models/CustomVoiceUpdate.swift

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+
+public struct CustomVoiceUpdate: JSONEncodable {
+    
+    /// The new name for the custom voice model.
+    public let name: String?
+    
+    /// The new description for the custom voice model.
+    public let description: String?
+    
+    /// A list of words and their translations from the custom voice model.
+    public let words: [Word]
+    
+    init(name: String? = nil, description: String? = nil, words: [Word]? = nil) {
```

Comment:
Two things here.  The init method should be public for full access to instantiation.  In addition, the words array should be set to a default of [] or emtpy array and remove the optional.  This better matches the words property array.